src/test/java/StringAddCalculator/StringAddCalculator.java

```diff
@@ -0,0 +1,42 @@
+package StringAddCalculator;
+
+public class StringAddCalculator {
+    public static int splitAndSum(String inText) {
+
+        if(inText == null || inText.isEmpty())
```

Comment:
전체적으로 코드 컨벤션 정리가 필요할 것 같습니다 😄 
```suggestion
        if (inText == null || inText.isEmpty())
```
요 내용 한번 보시면 좋을 것 같습니다 🙇 
> https://google.github.io/styleguide/javaguide.html#s4-formatting